pkg/controllers/disruption/controller.go

```diff
@@ -114,7 +114,7 @@ func (c *Controller) Reconcile(ctx context.Context, _ reconcile.Request) (reconc
 	// with making any scheduling decision off of our state nodes. Otherwise, we have the potential to make
 	// a scheduling decision based on a smaller subset of nodes in our cluster state than actually exist.
 	if !c.cluster.Synced(ctx) {
-		logging.FromContext(ctx).Debugf("waiting on cluster sync")
+		controllerruntime.LoggerFrom(ctx).V(3).Info("waiting on cluster sync")
```

Comment:
Have we considered just directly going to the `controller-runtime` package where all of this log logic is written (e.g. "sigs.k8s.io/controller-runtime/pkg/log") and just calling the function with `log.FromContext`? Considering how often that we log, I'd prefer that we keep how much we have to type each time that we want to log each time to a minimum

Reply:
> Have we considered just directly going to the `controller-runtime` package where all of this log logic is written (e.g. "sigs.k8s.io/controller-runtime/pkg/log") and just calling the function with `log.FromContext`?

No, I missed that. Definitely makes sense to use [FromContext()](https://github.com/kubernetes-sigs/controller-runtime/blob/56159419231e985c091ef3e7a8a3dee40ddf1d73/pkg/log/log.go#L91) instead. Thanks for the suggestion.

Reply:
Fixed.

Reply:
Can we just call it `log`, rather than `ctrl`. Strikes me as a bit more self-explanatory about what the package is doing to say `log.FromContext()` rather than `ctrl.FromContext`

Reply:
Done.